src/Link.cc

```diff
@@ -44,6 +44,23 @@ class ignition::gazebo::LinkPrivate
 using namespace ignition;
 using namespace gazebo;
 
+//////////////////////////////////////////////////
+// Helper function to create and remove components to enable and disable
+// functionality.
+template <class ComponentType>
```

Comment:
I'm just wondering if it would make sense to make this API public. There are many use cases for `enableComponent<T>` apart from just `LinearVelocity`. I'm not sure where the best place is to put it but this pattern is prevalent throughout almost all plugins. It'd be far cleaner to just expose this.

Reply:
I agree, this may make sense to expose in `utils` or the like.

Reply:
Moved it to `Util.hh` in 9f61ff885ad48bc7c010a9ef964e2fbf31f2d062. Outside of the `Link` class, the `enable` word may not make much sense though. I tried to explain in the docs because I couldn't think of a better name.